package.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Necessary run common commands like `build`, `development` should be in `package.json` inside \`scripts'. Here is a basic example